### ISSUE TITLE
Summary line of the Account Display is "odd"

### ISSUE THREAD

**Issue by grumpy235** [moneymanagerex/moneymanagerex#3669]
MMEX version:
 - [X ] 1.5.x
 - [ ] 1.6.x
Bug appears on 1.5.3 and master at 7501c44f

Operating System:
 - [X ] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Had a quick search and could not find this as already reported but apologies if it has.
Summary line of the Account Display is "odd"
![P1](https://user-images.githubusercontent.com/24897056/123780772-73576480-d8cb-11eb-9506-3b41c5aa80ba.PNG)

Not exactly certain what "Displayed Bal" is meant to show but I guess it is calculated from the 3 previous numbers. The last action on this account was a transfer out of 10K which has not yet been reconciled.

As an aside compiling mmcheckingpanel.cpp gives the following warning using VS2019
![P2](https://user-images.githubusercontent.com/24897056/123781850-98000c00-d8cc-11eb-80a3-0e761af01a94.PNG)

# Reproduction

Is the bug reproducible? 
 - [X ] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. 
2. 
3. 

…

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by vomikan** [comment]
it's balance of the displayed transactions in case if some filter has been applied.

**Comment by grumpy235** [comment]
@vomikan I am not using a filter so I guess in my case it is meaningless ? My displayed balance is 20,608.95 (the last line on my display) not 608.95

**Comment by whalley** [comment]
@grumpy235 What do you suggest the label should say?

**Comment by grumpy235** [comment]
@whalley Good question :grin: The moment I wrote that I realised it was not much help and thinking about it now, knowing what it is meant to convey, I am still not sure. My confusion was that the other 3 numbers were obvious, they could be seen in the last line of the display but the number in question appeared to be conjured out of nowhere.
Some ideas..
1. After a brief look, I could not find mention of it in the Help file so perhaps an explanation there
2. When you hover the mouse over the value then a balloon with a brief explanation of what it means
3. Put the name of the filter in the text so in my case "Displayed Bal:" would become "Last 12 Months Bal:" - not sure how this would work with Advanced Filter though
